tests/unit/test_dirs.py

```diff
@@ -76,7 +76,9 @@ def test_get_stage_dir_with_partitions(partitions):
 @pytest.mark.usefixtures("enable_partitions_feature")
```

Comment:
I suspect the test just above, `test_get_stage_dir_with_partitions` may fail in the same way, so I suggest applying the same fix.